source/_includes/site/header.html

```diff
@@ -40,6 +40,15 @@
               </li>
               <li>
                 <a href="/dashboards/">Dashboards</a>
+              </li>              
+              <li>
+                <a href="/docs/assist">Assist</a>
+              </li>
+              <li>
+                <a href="https://yellow.home-assistant.io/">Home Assistant Yellow</a>
+              </li>
+              <li>
+                <a href="https://skyconnect.home-assistant.io/">Home Assistant SkyConnect</a>
```

Comment:
I do get the idea, and understand the issue. I do think this might not be the best place. Mostly, because this is now a main menu item navigating away from the website, which is (IMHO) very confusing.